lib/identity-api-client/actions.rb

```diff
@@ -61,8 +61,9 @@ def recently_updated(from_time: nil, to_time: nil, hours_ago: nil)
       end
     end
 
-    def recently_taken(from_time: nil, to_time: nil, hours_ago: nil)
+    def recently_taken(from_time: nil, to_time: nil, hours_ago: nil, only_unique_members: nil)
       query_params = recent_api_query_params(from_time, to_time, hours_ago)
+      query_params.merge!({only_unique_members: 'true'}) if only_unique_members
```

Comment:
Pending the investigation on the Identity API (is it used by anything else?) we may be able to avoid adding this new param.